client/signup/jetpack-connect/new-site/index.jsx

```diff
@@ -0,0 +1,115 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import config from 'config';
+import { recordTracksEvent } from 'state/analytics/actions';
+import SiteURLInput from '../site-url-input';
+import ReaderBack from '../../../blocks/reader-full-post/back';
+
+import WordPressLogo from './wordpress-logo';
+import JetpackLogo from './jetpack-logo';
+
+const JetpackNewSite = React.createClass( {
+	displayName: 'JetpackNewSite',
+
+	componentDidMount() {
+		this.props.recordTracksEvent( 'calypso_jetpack_new_site_view' );
+	},
+
+	getNewWpcomSiteUrl() {
+		return config( 'signup_url' ) + '?ref=calypso-selector';
+	},
+
+	onURLEnter() {
+		this.props.recordTracksEvent( 'calypso_jetpack_new_site_connect_click' );
+		page( '/jetpack/connect?url=' + this.refs.siteUrlInputRef.state.value.toLowerCase() );
+	},
+
+	handleOnClickTos() {
+		this.props.recordTracksEvent( 'calypso_jpc_tos_link_click' );
+	},
+
+	handleBack( event ) {
+		event.preventDefault();
+
+		// @todo
+		// - Record action that user went back
+		// - Go back in history
+	},
+
+	render() {
+		return (
+			<div>
+				<ReaderBack onBackClick={ this.handleBack } />
+				<div className="jetpack-new-site">
+					<div className="jetpack-new-site__header">
+						<h2 className="jetpack-new-site__header-title">{ this.translate( 'Add a New Site' ) }</h2>
+						<div className="jetpack-new-site__header-text">{ this.translate(
+							'Create a new site on WordPress.com or add your existing self-hosted WordPress site with Jetpack.'
+						) } </div>
+					</div>
+					<div className="jetpack-new-site__content">
+						<Card className="jetpack-new-site__wpcom-site">
+							<WordPressLogo />
+							<h3 className="jetpack-new-site__card-title">
+								{ this.translate( 'Create a new shiny WordPress.com site' ) }
+							</h3>
+							<div className="jetpack-new-site__card-description">
+							<p>
+								{ this.translate( 'Start telling your story in just 2 minutes. Pick a visual theme and a domain — we’ll take care of the entire setup. If you need help we’ve got you covered with 24/7 support.' ) }
+							</p>
+							</div>
+							<div className="jetpack-new-site__button-holder">
+								<Button className="button is-primary" href={ this.getNewWpcomSiteUrl() }>{ this.translate( 'Start Now' ) }</Button>
```

Comment:
Hi! I've found a possible matching string that has already been translated **22** times:
`translate( 'Start now' )` *ES Score: 11*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/try/jetpack-sites-new)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).